csug/syntax.stex

```diff
@@ -709,6 +712,29 @@ Attaching a new property with the same name as an property already
 attached to a binding shadows the existing property with the new
 property.
 
+%----------------------------------------------------------------------------
+\entryheader
+\formdef{property-value}{\categoryprocedure}{(property-value \var{identifier} \var{key-identifier})}
+\formdef{property-value}{\categoryprocedure}{(property-value \var{identifier} \var{key-identifier} \var{default})}
+\returns see below
+\listlibraries
+\endentryheader
+
+The \scheme{property-value} procedure returns the value of the
+\var{key-identifier} property of \var{identifier}.
+If \var{identifier} has no \var{key-identifier} property, \var{default}
+is returned.
+If omitted, \var{default} defaults to \scheme{#f}.
+As the right-hand side of a \scheme{define-property} form is evaluated
+at expand-time, note that invoking \scheme{property-value} at relative
+run-time within the same library or top-level program won't see the
+property.
+This is the reason why the \scheme{get-property} macro wrapper around
+\scheme{property-value} is used below.
```

Comment:
Delete this section. I tried to include the essence of the content in the revision above.